src/app/hooks/useAI.ts

```diff
@@ -0,0 +1,14 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+
+import { createResponse } from '../services/chatbox';
+
+export const useAI = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationFn: createResponse,
```

Comment:
Ten generic qua, can dat ten cu the hon, vi du neu a doc se k hieu createResponse la gi